sample/SampleHost/SampleHost.csproj

```diff
@@ -15,7 +15,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.EventHubs" Version="3.0.6" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.0-beta.5" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.0-beta.3" />
```

Comment:
What's going on here - why are we decrementing this?

Reply:
I am fixing the versions in the tests/sample as current version is [5.0.0-beta.2](https://github.com/Azure/azure-webjobs-sdk/blob/dev/build/common.props#L5)